### ISSUE TITLE
clicking on nzbget icon in results isn't working

### ISSUE THREAD

**Issue by mauirixxx** [theotherp/nzbhydra#233]
I have 2 downloaders setup, and when I get a result from a search and click the nzbget icon, I get an infinite swirl like so: https://i.imgur.com/5QQDb0C.png

Here's the relevant log (domain name edited out):

2016-05-10 10:40:15,137 - ERROR - web - The client encountered the following error: TypeError: Cannot set property '#<Object>' of undefined
at http://www.example.net:5075/static/js/nzbhydra.js:4721:44
at processQueue (http://www.example.net:5075/static/js/alllibs.js:24763:28)
at http://www.example.net:5075/static/js/alllibs.js:24779:27
at Scope.$eval (http://www.example.net:5075/static/js/alllibs.js:26031:28)
at Scope.$digest (http://www.example.net:5075/static/js/alllibs.js:25847:31)
at Scope.$apply (http://www.example.net:5075/static/js/alllibs.js:26139:24)
at done (http://www.example.net:5075/static/js/alllibs.js:20477:47)
at completeRequest (http://www.example.net:5075/static/js/alllibs.js:20675:7)
at XMLHttpRequest.requestLoaded (http://www.example.net:5075/static/js/alllibs.js:20616:9). Caused by: No known cause

And it just repeats this bit of log if I click on the nzbget of any other results. I have both downloaders configured as so: https://i.imgur.com/vYnjhum.png

**Comment by theotherp** [comment]
And here I was thinking that for one my testing a new feature would be enough. 

Thanks, won't be able to fix this before thursday.

**Comment by theotherp** [comment]
Does it work for the other one? What if you disable one?

**Comment by theotherp** [comment]
By the way, do the downloader names contain hashes or any other special characters?

**Comment by mauirixxx** [comment]
I came across another bug while testing this one. First things first - I disabled the first downloader I entered, and searched, and clicked - same results.

Now the bug - I can't re-enable a disabled downloader, either from the main "add a downloader" page or from with in the downloader configuration page either - the slider just refuses to move. I don't see anything in the log file about it.

Downloader names are as follows: work-nzbget & don-nzbget
